auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -365,30 +400,103 @@ def get_augment_transforms(self):
         if self.roi_size is None:
             raise ValueError("roi_size is not specified")
 
+        augment_mode = self.augment_params.get("augment_mode", None)
+        augment_flips = self.augment_params.get("augment_flips", None)
+        augment_rots = self.augment_params.get("augment_rots", None)
+
+        if self.debug:
+            print(f"Using augment_mode {augment_mode}, augment_flips {augment_flips}  augment_rots {augment_rots}")
+
+
         ts = []
-        ts.append(
-            RandAffined(
-                keys=[self.image_key, self.label_key],
-                prob=0.2,
-                rotate_range=[0.26, 0.26, 0.26],
-                scale_range=[0.2, 0.2, 0.2],
-                mode=["bilinear", "nearest"],
-                spatial_size=self.roi_size,
-                cache_grid=True,
-                padding_mode="border",
+
+        if augment_mode is None or augment_mode=='default':
+
+            ts.append(
+                RandAffined(
+                    keys=[self.image_key, self.label_key],
+                    prob=0.2,
+                    rotate_range=[0.26, 0.26, 0.26],
+                    scale_range=[0.2, 0.2, 0.2],
+                    mode=["bilinear", "nearest"],
+                    spatial_size=self.roi_size,
+                    cache_grid=True,
+                    padding_mode="border",
+                )
             )
-        )
-        ts.append(RandFlipd(keys=[self.image_key, self.label_key], prob=0.5, spatial_axis=0))
-        ts.append(RandFlipd(keys=[self.image_key, self.label_key], prob=0.5, spatial_axis=1))
-        ts.append(RandFlipd(keys=[self.image_key, self.label_key], prob=0.5, spatial_axis=2))
-        ts.append(
-            RandGaussianSmoothd(
-                keys=self.image_key, prob=0.2, sigma_x=[0.5, 1.0], sigma_y=[0.5, 1.0], sigma_z=[0.5, 1.0]
+            ts.append(
+                RandGaussianSmoothd(
+                    keys=self.image_key, prob=0.2, sigma_x=[0.5, 1.0], sigma_y=[0.5, 1.0], sigma_z=[0.5, 1.0]
+                )
+            )
+            ts.append(RandScaleIntensityd(keys=self.image_key, prob=0.5, factors=0.3))
+            ts.append(RandShiftIntensityd(keys=self.image_key, prob=0.5, offsets=0.1))
+            ts.append(RandGaussianNoised(keys=self.image_key, prob=0.2, mean=0.0, std=0.1))
+
+        elif augment_mode=='none':
+
+            augment_flips=[]
+            augment_rots=[]
```

Comment:
Looks like can be removed?
```suggestion
```

Reply:
no, we need it to set them as empty lists